packages/lesswrong/server/resolvers/postResolvers.ts

```diff
@@ -242,7 +243,31 @@ augmentFieldsDict(Posts, {
         return getDialogueMessageContents(post, dialogueMessageId)
       }
     }
-  }
+  },
+  annualReviewMarketProbability: {
+    resolveAs: {
+      type: 'Float',
+      resolver: async (post: DbPost, args: void, context: ResolverContext) => {
+        return (await getPostMarketInfo(post))?.probability
+      }
+    }
+  },
+  annualReviewMarketIsResolved: {
+    resolveAs: {
+      type: 'Boolean',
+      resolver: async (post: DbPost, args: void, context: ResolverContext) => {
+        return (await getPostMarketInfo(post))?.isResolved
+      }
+    }
+  },
+  annualReviewMarketYear: {
+    resolveAs: {
+      type: 'Int',
+      resolver: async (post: DbPost, args: void, context: ResolverContext) => {
+        return (await getPostMarketInfo(post))?.year
```

Comment:
I think these probably want to use `getWithCustomLoader` so that we can make one request instead of 3 for the same info

Reply:
What's the caching behaviour of `getWithCustomLoader`?